src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -264,6 +263,34 @@ public Measure updateMeasure(
     return measureRepository.save(outputMeasure);
   }
 
+  public Measure deactivateMeasure(final Measure existingMeasure, final String username) {
+    final String id = existingMeasure.getId();
```

Comment:
This will error if existingMeasure is null. Suggest moving below the null check.